x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/common/field_value.tsx

```diff
@@ -30,9 +31,25 @@ export interface IndicatorFieldValueProps {
  * Takes an indicator object, a field and a field => type object to returns the correct value to display.
  * @returns If the type is a 'date', returns the {@link DateFormatter} component, else returns the value or {@link EMPTY_VALUE}.
  */
```

Comment:
Let's move this comment down next to the `IndicatorFieldValue` and update the description a bit to reflect the changes added by this PR